defaults/main.yml

```diff
@@ -6,7 +6,8 @@
 #
 
 # package name (version)
-ufw_package: ufw
+ufw_packages:
```

Comment:
Is there any reason why you need to define more than one package here? 
Changing this would break backwards compatibility

Reply:
I forgot to mention, in Archlinux the ufw package is split into ufw and ufw-extras. The extra package contains the application templates for ufw. For me it made sense to install and use both so I changed it to an array of packages.
Otherwise we could add an optional variable ufw_extra_packages or something?

Reply:
This might be a good solution for the time being

```yaml
# Depricated: this will be removed in future releases! Use `ufw_packages`
ufw_package: ufw
# added to support systems where ufw packages are split up
ufw_packages:
  - "{{ ufw_package }}"
```

Reply:
okay fixed